pandas/core/tools/datetimes.py

```diff
@@ -726,13 +729,30 @@ def to_datetime(
             with year first.
 
     utc : bool, default None
-        Return UTC DatetimeIndex if True (converting any tz-aware
-        datetime.datetime objects as well).
+        Control timezone-related parsing, localization and conversion.
+
+        - If ``True``, the function *always* returns a timezone-aware
+          UTC-localized Timestamp, Series or DatetimeIndex. To do this,
+          timezone-naive inputs are *localized* as UTC (e.g.
+          ``01-01-2020 01:00:00`` becomes ``01-01-2020 01:00:00Z``), while
```

Comment:
I think the `e.g.` examples here should be includes in the `Examples` section to simplify the description here.

Reply:
ok